NeurodataWithoutBorders/pynwb
# Correct README so that it validates correctly

## Motivation

The README.rst did not validate as is, which would cause a problem on PyPi. I removed an unnecessary table directive. While fixing this issue I also corrected some typos in the CONTRIBUTING.rst.

Fixes #238 .

## How to test the behavior?
```
$ pip install readme
$ python setup.py check -r -s
```
This will now return no warnings.

## Checklist

- [x] Is your contribution compliant with out coding style ? This can be checked running `flake8` from the source directory.
